js/core/transform-json-response.ts

```diff
@@ -44,6 +44,7 @@ export interface IPortalData {
     activityUrl: string;
     id: number;
     teacher: string;
+    hasTeacherEdition: boolean;
```

Comment:
How is this value determined?  Is this on a per activity basis?  So we're not going to always show the teacher edition button, but only show it if think an activity has teacher edition content? Am I understanding this correctly?

Reply:
This is passed in through the offering data the portal receives. So I added it to the mock data.

![image](https://user-images.githubusercontent.com/7716653/93802533-71ec6380-fbf8-11ea-821c-f90f548c0986.png)

Reply:
Kiley said button should only be shown when there is teacher edition is available.